dev/plugins/hu.elte.txtuml.xtxtuml/src/hu/elte/txtuml/xtxtuml/XtxtUML.xtext

```diff
@@ -212,10 +212,18 @@ TUMultiplicity :
 
 // Commons
 
+TUModifiers:
+	{TUModifiers} (static?='static'? & visibility=TUVisibility? & externality=TUExternality?)
```

Comment:
Try to remove `{TUModifiers}`. Result: Xtext warning.

> The rule 'TUModifiers' may be consumed without object instantiation. Add an action to ensure object creation, e.g. '{TUModifiers}'.

Try to remove the parentheses. Result: Xtext error.

> Actions may not be used in unordered groups.